policyengine_us/parameters/gov/usda/snap/income/deductions/utility/standard/by_household_size.yaml

```diff
@@ -0,0 +1,32 @@
+description: State standard utility allowance broken down by household size
+NC:
+  1:
+    2024-01-01: 585
```

Comment:
was this introduced in 2024? Or was this been in place historically?

Reply:
Regardless we should use a bool parameter indicating the transition for a flat utility allowance structure to one by household size

Reply:
Any suggestions on how that structure might look like?

Reply:
Thinking for checking if the state is in the created parameter list and if so then return the value of the newly created variable that includes the household breakdown - lmk if that makes sense

Reply:
I was thinking that we would just modify the household_size_states.yaml file. i.e. if the state no longer is household size dependent:

```yaml
values:
  2024-01-01:
    - NC
  2028-01-01: []
```

Reply:
something like this?
```
description: The following states allow for a different SNAP utility deduction, based on household size. 
NC:
  2024-01-01: true

metadata:
  unit: bool
  label: States where the utility deduction is based off of household size
  period: year
  breakdown:
  - state_code
```

in 

> policyengine_us\parameters\gov\usda\snap\income\deductions\utility\standard\by_household_size\participating_states.yaml